index.php

```diff
@@ -1,31 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
+
 <body>
-    
-<?php
 
-use Collection\Monitors\MonitorModels;
+    <?php
+
+    use Collection\Monitors\MonitorModels;
+
+    require_once 'vendor/autoload.php';
+
+    // Getting connected
+    $db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
-require_once 'vendor/autoload.php';
+    $model = new MonitorModels($db);
 
-// Getting connected
-$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $monitors = $model->getAllMonitors();
 
-$model = new MonitorModels($db);
+    foreach ($monitors as $monitor) {
+        echo "$monitor->make\n" . "$monitor->model\n" . $monitor->commissioned . "<br>";
+    }
 
-$monitors = $model->getAllMonitors();
+    ?>
 
-foreach ($monitors as $monitor) {
-    echo "$monitor->make\n"."$monitor->model\n".$monitor->commissioned."<br>";
-}
+    <form method="POST">
+        <label for="make">Make:</label>
+        <input type="text" id="make" name="make">
+        <label for="model">Model:</label>
+        <input type="text" id="model" name="model">
+        <label for="commissioned">Commissioned:</label>
```

Comment:
You should add some more info here to tell users what format the date needs to be in

Reply:
Added date format to label